RFC-000-rfc-process-suggestions.md

```diff
@@ -0,0 +1,77 @@
+# RFC Zero - Open for suggestions to the RFC process
+
+## Proposed by
+
+[Alyssa Rock](https://thegooddocs.slack.com/team/U012KCMPP0V)
+
+## Current status
+
+- [x] Draft
+- [x] Under discussion (until 2021-07-28)
+- [ ] Final comment and voting (until YYYY-MM-DD) {{Add date after selecting this status.}}
+- [ ] Accepted
+- [ ] Rejected
+- [ ] Implemented
+- [ ] Deferred
+- [ ] Withdrawn
+
+
+## Proposal overview

```

Comment:
👌🏻 for Proposal Overview, not sure what the default text is here.  Recommend we have a suggestion to keep it short 😉

Reply:
I agree. I've added your suggestion to the RFC.